extensions/amp-analytics/amp-analytics.md

```diff
@@ -16,79 +16,135 @@ limitations under the License.
+when the document is first loaded, and each time an `<a>` tag is clicked:
 
-The tag is driven by JSON config that can be specified in one of the three places:
-
-**Remote configs**: This config is loaded in the page via a URL. The URL can be specified in the `config` attribute of the `amp-analytics` tag.
-**Inline configs**: This config is specified directly on the AMP page inside the `amp-analytics` tag.
-**Built in configs**: The AMP platform has these configs built into the platform. These can be loaded onto the page via the `type` attribute on the `amp-analytics` tag.
-
-If more than one methods of specifying the config are used, the configs are merged together in the order above (highest to lowest precedence).
-
-
-##### <a name="inline"></a>Inline Config with a custom host & url format
-```javascript
+```html
 <amp-analytics>
 <script type="application/json">
 {
-  "host": "my-analytics.com:8080",
   "requests": {
-    "base_hit": "/collect?v=1&_v=a0&aip=true&_s=HIT_COUNT&dl=DOMAIN&dt=TITLE&sr=SCREEN_WIDTHxSCREEN_HEIGHT&ht=TIMESTAMP&cid=CLIENT_IDENTIFIER&tid=ACCOUNT"
-    "pageview": "/r/{base_hit}&t=pageview&_r=1"
-  }
+    "pageview": "https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}",
+    "event": "https://example.com/analytics?eid=${eventId}&elab=${eventLabel}&acct=${account}"
+  },
   "vars": {
-    "account_id": "123456",
+    "account": "ABC123"
+  },
+  "triggers": {
+    "track pageview": {
+      "on": "visible",
+      "request": "pageview"
+    },
+    "track anchor clicks": {
+      "on": "click",
+      "selector": "a",
+      "request": "event",
+      "vars": {
+        "eventId": "42",
+        "eventLabel": "clicked on a link"
+      }
+    }
   }
-  "triggers": [{
-    "on": "LOAD",
-    "request": "pageview"
-  }]
 }
 </script>
 </amp-analytics>
 ```
-##### <a name="builtin"></a>Built-in tag for a service `XYZ` with inline config
+
+##### Configuration
+
+Configuration may be specified inline (as shown in the example above) or fetched remotely by specifying a URL in the
+`config` attribute. Additionally, built-in configuration for popular analytics vendors can be selected using 
+the `type` attribute. 
+
+If configuration data from more than one of these sources is used, the configuration objects will
+be merged together such that remote configuration takes precedence over inline configuration, and inline configuration
+takes precendence over vendor configuration.
+
+The `<amp-analytics>` configuration object uses the following format:
 
 ```javascript
-<amp-analytics type="xyz">
-<script type="application/json">
 {
-  "vars": { "account_id": "123456" }
-  "triggers": [{
-    "on": "LOAD",
-    "request": "pageview"
-  }]
+  "requests": { *request-name*: *request-value*, ... },
+  "vars": { *var-name*: *var-value*, ... },
+  "triggers": { *trigger-name*: *trigger-object*, ...},
+  "transport": { beacon: *boolean*, xhrpost: *boolean*, image: *boolean* }
```

Comment:
Let's format property names as correct JSON - e.g. using "beacon" and all others.

Also - the space is cheap here, let's add line breaks :)